scenes/game/local_game.gd

```diff
@@ -4859,6 +4861,12 @@ func continue_resolve_strike(events):
 				events += player1.cleanup_continuous_boosts()
 				events += player2.cleanup_continuous_boosts()
 
+				# If hit, move card to gauge, otherwise move to discard.
+				if card1 in active_strike.cards_in_play:
+					events += strike_send_attack_to_discard_or_gauge(player1, card1)
+				if card2 in active_strike.cards_in_play:
+					events += strike_send_attack_to_discard_or_gauge(player2, card2)
+
```

Comment:
want to just put in an assert after this that active_strike.cards_in_play.size() == 0?

Can you also add comments above to explain that handle_strike_attack_cleanup() handles all effects aside from the standard hit/miss and that those are handled down here because boosts go to discard first?